README.md

```diff
@@ -45,7 +45,7 @@ nginx supervisor postgresql libpq-dev npm nodejs-legacy libhdf5-dev libnetcdf-de
 
 2. Install Python dependencies: `pip install -r requirements.txt`
 
-3. Initialize the RethinkDB database with `make db_init`
+3. Initialize the database with `make db_init`
```

Comment:
This should be swapped with the current step 4.  Also, users may need to restart postgresqld at this point.